### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by mcovalt**
To prepare for ~Independence~ Linting Day I'm starting this Draft PR.

TL;DR:
- `manage.py lint`: format ([Black](https://github.com/python/black)) then lint ([pyflakes](https://github.com/PyCQA/pyflakes)) the _modified_ files in the repo
- `manage.py lint --all`: format then lint _all_ files in the repo
- `manage.py lint --no-fix`: lint without formatting
- `manage.py lint --no-lint`: format without linting

Tests are included so that linting errors and non-formatted code will cause a failed test. Which is why this is a Draft PR. Lots of linting errors need to be fixed.

~I suggest we address linting errors on this branch before formatting. That way upstream changes from `dev` can be merged in cleanly. Then we can format right before we are ready to merge into `dev`.~
EDIT: we decided to lint as we go.

### CODE REVIEW

**Comment by debboutr** [comment]
this looks really good, I suppose we can just live w/ the linting/Black errors in the test while we work to clean them up, this is going to make fixing conflicts in merges a whole lot easier.
```
FAIL: test_pyflakes (dashboard.tests.unit.test_lint.LintTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/debboutr/projects/factotum/dashboard/tests/unit/test_lint.py", line 38, in test_pyflakes
    self.assertFalse(not_linted, "Found files with linting errors.")
AssertionError: True is not false : Found files with linting errors.
```

**Comment by mcovalt** [comment]
The big formatting has been pushed up. In order to merge this branch into your working branch without grabbing the formats, use `git merge 7ee84a8`.
